docs/contributing.md

```diff
@@ -78,9 +79,16 @@ Check linting rules are passing:
 ```shell
 make checks-validator
 ```
-
 lakeFS uses [go fmt](https://golang.org/cmd/gofmt/) as a style guide for Go code.
 
+Run system-tests:
+```shell
+make system-tests
+```
+
+Want to dive deeper into our system tests infrastructure? Need to debug the tests? Follow [this](https://github.com/treeverse/lakeFS/blob/master/esti/README.md) documentation
```

Comment:
You are right, the only solution I see is to place this document in the repository root and change the name to "system_tests.md" WDYT? Note that this will not solve the issue of the assets (images) still under the esti path (we can take them out as well but then it will just spam the repository root

Reply:
Nothing is actionable - just information, the options. Still prefer the link you wrote to master.